plugins/VstEffect/VstEffectControlDialog.h

```diff
@@ -62,6 +62,9 @@ class VstEffectControlDialog : public EffectControlDialog
 	VstPlugin * m_plugin;
 
 	QLabel * tbLabel;
+
+private slots:
+	void togglePluginUI( bool checked );
```

Comment:
You should use `const` only when it is useful:
* Pointers and references.
* Read-only methods.
* Constants.

As I said, only when it is useful, not when it is possible. Excessive use of `const` makes the code harder to read (this is subjective, but it is simpler). Without `const`, you can reuse parameters (simplicity) and you do not have to take `const` back if you need write access later (less typing and less thinking).

 `const bool` and alike are useless in prototypes because values are copied. Do you see `const` in the `dup2` prototype?

It used to be a good idea to add `const` to avoid `if (x = 5)` errors, but a good compiler with warnings will warn you. I will not ask you to remove `const` in a complex function body, but you should not ask others to use `const` unless there is a clear benefit.